frontend/components/commentWriter.tsx

```diff
@@ -196,25 +205,29 @@ const CommentWriter: React.FC<CommentWriterProps> = ({ propId }) => {
               },
             }
           );
+
           toast.success("Proof submitted successfully!", {
             position: "bottom-right",
           });
+          refetch();
```

Comment:
new comment is added so we want to refetch the data from the server to populate it